avro-schema/sum-part.avsc

```diff
@@ -4,10 +4,15 @@
     "name": "PrioSumPart",
     "fields": [
         {
-            "name": "batch_uuid",
-            "type": "string",
-            "logicalType": "uuid",
-            "doc": "Universal unique identifier to link with data share batch sent to other server(s) participating in the aggregation."
+            "name": "batch_uuids",
+            "type": {
+                "type": "array",
+                "items": {
+                    "type": "string",
+                    "logicalType": "uuid"
+                }
+            },
+            "doc": "Universal unique identifiers of data share batches included in this sum."
```

Comment:
Abbreviate to "UUID" for consistency with other comments and to make it easier to read.